CRITICAL - 2023-04-30 11:40:13 --> Undefined variable $route
in APPPATH\Config\Routes.php on line 41.
 1 APPPATH\Config\Routes.php(41): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $route', 'C:\\xampp\\htdocs\\ci4\\project-root\\app\\Config\\Routes.php', 41)
 2 SYSTEMPATH\Router\RouteCollection.php(267): require('C:\\xampp\\htdocs\\ci4\\project-root\\app\\Config\\Routes.php')
 3 SYSTEMPATH\CodeIgniter.php(802): CodeIgniter\Router\RouteCollection->loadRoutes()
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 11:49:00 --> Undefined variable $route
in APPPATH\Config\Routes.php on line 41.
 1 APPPATH\Config\Routes.php(41): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $route', 'C:\\xampp\\htdocs\\ci4\\project-root\\app\\Config\\Routes.php', 41)
 2 SYSTEMPATH\Router\RouteCollection.php(267): require('C:\\xampp\\htdocs\\ci4\\project-root\\app\\Config\\Routes.php')
 3 SYSTEMPATH\CodeIgniter.php(802): CodeIgniter\Router\RouteCollection->loadRoutes()
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 14:58:09 --> Class "App\Controllers\CI_Controller" not found
in APPPATH\Controllers\DevisController.php on line 6.
 1 SYSTEMPATH\Autoloader\Autoloader.php(312): include_once()
 2 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ci4\\project-root\\app\\Controllers\\DevisController.php')
 3 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\DevisController')
 4 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\DevisController')
 5 SYSTEMPATH\CodeIgniter.php(892): class_exists('\\App\\Controllers\\DevisController', true)
 6 SYSTEMPATH\CodeIgniter.php(486): CodeIgniter\CodeIgniter->startController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 14:58:53 --> Class "App\Controllers\CI_Controller" not found
in APPPATH\Controllers\DevisController.php on line 6.
 1 SYSTEMPATH\Autoloader\Autoloader.php(312): include_once()
 2 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ci4\\project-root\\app\\Controllers\\DevisController.php')
 3 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\DevisController')
 4 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\DevisController')
 5 SYSTEMPATH\CodeIgniter.php(892): class_exists('\\App\\Controllers\\DevisController', true)
 6 SYSTEMPATH\CodeIgniter.php(486): CodeIgniter\CodeIgniter->startController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 15:40:53 --> Undefined variable $route
in APPPATH\Config\Routes.php on line 36.
 1 APPPATH\Config\Routes.php(36): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $route', 'C:\\xampp\\htdocs\\ci4\\project-root\\app\\Config\\Routes.php', 36)
 2 SYSTEMPATH\Router\RouteCollection.php(267): require('C:\\xampp\\htdocs\\ci4\\project-root\\app\\Config\\Routes.php')
 3 SYSTEMPATH\CodeIgniter.php(802): CodeIgniter\Router\RouteCollection->loadRoutes()
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 15:40:54 --> Undefined variable $route
in APPPATH\Config\Routes.php on line 36.
 1 APPPATH\Config\Routes.php(36): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $route', 'C:\\xampp\\htdocs\\ci4\\project-root\\app\\Config\\Routes.php', 36)
 2 SYSTEMPATH\Router\RouteCollection.php(267): require('C:\\xampp\\htdocs\\ci4\\project-root\\app\\Config\\Routes.php')
 3 SYSTEMPATH\CodeIgniter.php(802): CodeIgniter\Router\RouteCollection->loadRoutes()
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 16:04:33 --> Too few arguments to function App\Controllers\DevisController::insert_devis(), 0 passed in C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php on line 934 and exactly 1 expected
in APPPATH\Controllers\DevisController.php on line 27.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DevisController->insert_devis()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 16:13:11 --> Object of type stdClass is not callable
in APPPATH\Controllers\DevisController.php on line 30.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DevisController->insert_devis()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 17:08:45 --> syntax error, unexpected token ":"
in APPPATH\Config\Routes.php on line 41.
 1 SYSTEMPATH\CodeIgniter.php(802): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 20:04:31 --> Object of class stdClass could not be converted to string
in SYSTEMPATH\Database\Query.php on line 335.
 1 SYSTEMPATH\Database\Query.php(335): strtr('INSERT INTO `services` (`id_devis`, `created_at`, `updated_at`) VALUES (:id_devis:, :created_at:, :updated_at:)', [...])
 2 SYSTEMPATH\Database\Query.php(310): CodeIgniter\Database\Query->matchNamedBinds('INSERT INTO `services` (`id_devis`, `created_at`, `updated_at`) VALUES (:id_devis:, :created_at:, :updated_at:)', [...])
 3 SYSTEMPATH\Database\Query.php(159): CodeIgniter\Database\Query->compileBinds()
 4 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\Query->getQuery()
 5 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `services` (`id_devis`, `created_at`, `updated_at`) VALUES (:id_devis:, :created_at:, :updated_at:)', [...], false)
 6 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 7 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 8 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 9 APPPATH\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
ERROR - 2023-04-30 20:07:36 --> mysqli_sql_exception: Unknown column 'prix' in 'field list' in C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `se...', 0)
#1 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `se...')
#2 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `se...')
#3 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `se...', Array, false)
#4 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci4\project-root\app\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
#8 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
#9 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
#10 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci4\project-root\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-04-30 20:07:36 --> Unknown column 'prix' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `services` (`prix`, `created_at`, `updated_at`) VALUES (\'20\', \'2023-04-30 20:07:36\', \'2023-04-30 20:07:36\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `services` (`prix`, `created_at`, `updated_at`) VALUES (\'20\', \'2023-04-30 20:07:36\', \'2023-04-30 20:07:36\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `services` (`prix`, `created_at`, `updated_at`) VALUES (\'20\', \'2023-04-30 20:07:36\', \'2023-04-30 20:07:36\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `services` (`prix`, `created_at`, `updated_at`) VALUES (:prix:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
ERROR - 2023-04-30 20:07:54 --> mysqli_sql_exception: Unknown column 'prix' in 'field list' in C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `se...', 0)
#1 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `se...')
#2 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `se...')
#3 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `se...', Array, false)
#4 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci4\project-root\app\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
#8 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
#9 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
#10 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci4\project-root\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-04-30 20:07:54 --> Unknown column 'prix' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `services` (`prix`, `created_at`, `updated_at`) VALUES (\'20\', \'2023-04-30 20:07:54\', \'2023-04-30 20:07:54\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `services` (`prix`, `created_at`, `updated_at`) VALUES (\'20\', \'2023-04-30 20:07:54\', \'2023-04-30 20:07:54\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `services` (`prix`, `created_at`, `updated_at`) VALUES (\'20\', \'2023-04-30 20:07:54\', \'2023-04-30 20:07:54\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `services` (`prix`, `created_at`, `updated_at`) VALUES (:prix:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 20:08:00 --> Object of class stdClass could not be converted to string
in SYSTEMPATH\Database\Query.php on line 335.
 1 SYSTEMPATH\Database\Query.php(335): strtr('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (:id_devis:, :prix:, :created_at:, :updated_at:)', [...])
 2 SYSTEMPATH\Database\Query.php(310): CodeIgniter\Database\Query->matchNamedBinds('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (:id_devis:, :prix:, :created_at:, :updated_at:)', [...])
 3 SYSTEMPATH\Database\Query.php(159): CodeIgniter\Database\Query->compileBinds()
 4 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\Query->getQuery()
 5 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (:id_devis:, :prix:, :created_at:, :updated_at:)', [...], false)
 6 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 7 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 8 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 9 APPPATH\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 20:08:25 --> Object of class stdClass could not be converted to string
in SYSTEMPATH\Database\Query.php on line 335.
 1 SYSTEMPATH\Database\Query.php(335): strtr('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (:id_devis:, :prix:, :created_at:, :updated_at:)', [...])
 2 SYSTEMPATH\Database\Query.php(310): CodeIgniter\Database\Query->matchNamedBinds('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (:id_devis:, :prix:, :created_at:, :updated_at:)', [...])
 3 SYSTEMPATH\Database\Query.php(159): CodeIgniter\Database\Query->compileBinds()
 4 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\Query->getQuery()
 5 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (:id_devis:, :prix:, :created_at:, :updated_at:)', [...], false)
 6 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 7 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 8 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 9 APPPATH\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
ERROR - 2023-04-30 20:08:52 --> mysqli_sql_exception: Unknown column 'id_devis' in 'field list' in C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `se...', 0)
#1 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `se...')
#2 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `se...')
#3 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `se...', Array, false)
#4 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci4\project-root\app\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
#8 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
#9 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
#10 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci4\project-root\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-04-30 20:08:52 --> Unknown column 'id_devis' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (9, \'35\', \'2023-04-30 20:08:52\', \'2023-04-30 20:08:52\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (9, \'35\', \'2023-04-30 20:08:52\', \'2023-04-30 20:08:52\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (9, \'35\', \'2023-04-30 20:08:52\', \'2023-04-30 20:08:52\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (:id_devis:, :prix:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
ERROR - 2023-04-30 20:15:12 --> mysqli_sql_exception: Unknown column 'id_devis' in 'field list' in C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `se...', 0)
#1 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `se...')
#2 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `se...')
#3 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `se...', Array, false)
#4 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci4\project-root\app\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
#8 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
#9 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
#10 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci4\project-root\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-04-30 20:15:12 --> Unknown column 'id_devis' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (10, \'35\', \'2023-04-30 20:15:12\', \'2023-04-30 20:15:12\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (10, \'35\', \'2023-04-30 20:15:12\', \'2023-04-30 20:15:12\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (10, \'35\', \'2023-04-30 20:15:12\', \'2023-04-30 20:15:12\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (:id_devis:, :prix:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
ERROR - 2023-04-30 20:15:12 --> mysqli_sql_exception: Unknown column 'id_devis' in 'field list' in C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `se...', 0)
#1 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `se...')
#2 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `se...')
#3 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `se...', Array, false)
#4 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci4\project-root\app\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
#8 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
#9 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
#10 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci4\project-root\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-04-30 20:15:12 --> Unknown column 'id_devis' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (11, \'35\', \'2023-04-30 20:15:12\', \'2023-04-30 20:15:12\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (11, \'35\', \'2023-04-30 20:15:12\', \'2023-04-30 20:15:12\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (11, \'35\', \'2023-04-30 20:15:12\', \'2023-04-30 20:15:12\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (:id_devis:, :prix:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
ERROR - 2023-04-30 20:19:57 --> mysqli_sql_exception: Unknown column 'id_devis' in 'field list' in C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `se...', 0)
#1 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `se...')
#2 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `se...')
#3 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `se...', Array, false)
#4 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci4\project-root\app\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
#8 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
#9 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
#10 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci4\project-root\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-04-30 20:19:57 --> Unknown column 'id_devis' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (1, 25, \'2023-04-30 20:19:57\', \'2023-04-30 20:19:57\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (1, 25, \'2023-04-30 20:19:57\', \'2023-04-30 20:19:57\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (1, 25, \'2023-04-30 20:19:57\', \'2023-04-30 20:19:57\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (:id_devis:, :prix:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
ERROR - 2023-04-30 20:19:57 --> mysqli_sql_exception: Unknown column 'id_devis' in 'field list' in C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `se...', 0)
#1 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `se...')
#2 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `se...')
#3 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `se...', Array, false)
#4 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci4\project-root\app\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
#8 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
#9 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
#10 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci4\project-root\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-04-30 20:19:57 --> Unknown column 'id_devis' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (1, 5, \'2023-04-30 20:19:57\', \'2023-04-30 20:19:57\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (1, 5, \'2023-04-30 20:19:57\', \'2023-04-30 20:19:57\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (1, 5, \'2023-04-30 20:19:57\', \'2023-04-30 20:19:57\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `services` (`id_devis`, `prix`, `created_at`, `updated_at`) VALUES (:id_devis:, :prix:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ServeurController.php(13): CodeIgniter\Model->insert(Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ServeurController->insert_serveur()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ServeurController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 21:24:46 --> Unclosed '{' on line 9
in APPPATH\Controllers\DevisController.php on line 48.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ci4\\project-root\\app\\Controllers\\DevisController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\DevisController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\DevisController')
 4 SYSTEMPATH\CodeIgniter.php(892): class_exists('\\App\\Controllers\\DevisController', true)
 5 SYSTEMPATH\CodeIgniter.php(486): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 21:25:27 --> Object of class stdClass could not be converted to string
in APPPATH\Controllers\DevisController.php on line 31.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DevisController->insert_devis()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 21:27:07 --> syntax error, unexpected token "}", expecting ";"
in APPPATH\Controllers\DevisController.php on line 47.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ci4\\project-root\\app\\Controllers\\DevisController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\DevisController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\DevisController')
 4 SYSTEMPATH\CodeIgniter.php(892): class_exists('\\App\\Controllers\\DevisController', true)
 5 SYSTEMPATH\CodeIgniter.php(486): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 21:27:15 --> syntax error, unexpected token "}", expecting ";"
in APPPATH\Controllers\DevisController.php on line 47.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ci4\\project-root\\app\\Controllers\\DevisController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\DevisController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\DevisController')
 4 SYSTEMPATH\CodeIgniter.php(892): class_exists('\\App\\Controllers\\DevisController', true)
 5 SYSTEMPATH\CodeIgniter.php(486): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 21:38:54 --> Cannot use object of type stdClass as array
in APPPATH\Controllers\DevisController.php on line 33.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DevisController->insert_devis()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 21:41:26 --> CodeIgniter\BaseModel::transformDataToArray(): Return value must be of type array, string returned
in SYSTEMPATH\BaseModel.php on line 1706.
 1 SYSTEMPATH\BaseModel.php(742): CodeIgniter\BaseModel->transformDataToArray('a', 'insert')
 2 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert('a', true)
 3 APPPATH\Controllers\DevisController.php(38): CodeIgniter\Model->insert('a', true, 'a')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DevisController->insert_devis()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 21:43:35 --> CodeIgniter\BaseModel::transformDataToArray(): Return value must be of type array, string returned
in SYSTEMPATH\BaseModel.php on line 1706.
 1 SYSTEMPATH\BaseModel.php(742): CodeIgniter\BaseModel->transformDataToArray('a', 'insert')
 2 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert('a', true)
 3 APPPATH\Controllers\DevisController.php(38): CodeIgniter\Model->insert('a', true, 'a')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DevisController->insert_devis()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 21:45:47 --> There is no data to insert.
in SYSTEMPATH\BaseModel.php on line 762.
 1 SYSTEMPATH\BaseModel.php(762): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([], true)
 3 APPPATH\Controllers\DevisController.php(44): CodeIgniter\Model->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DevisController->insert_devis()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 21:51:35 --> There is no data to insert.
in SYSTEMPATH\BaseModel.php on line 762.
 1 SYSTEMPATH\BaseModel.php(762): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([], true)
 3 APPPATH\Controllers\DevisController.php(44): CodeIgniter\Model->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DevisController->insert_devis()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 21:57:14 --> There is no data to insert.
in SYSTEMPATH\BaseModel.php on line 762.
 1 SYSTEMPATH\BaseModel.php(762): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([], true)
 3 APPPATH\Controllers\DevisController.php(44): CodeIgniter\Model->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DevisController->insert_devis()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 21:59:27 --> There is no data to insert.
in SYSTEMPATH\BaseModel.php on line 762.
 1 SYSTEMPATH\BaseModel.php(762): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([], true)
 3 APPPATH\Controllers\DevisController.php(45): CodeIgniter\Model->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DevisController->insert_devis()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
CRITICAL - 2023-04-30 22:01:54 --> There is no data to insert.
in SYSTEMPATH\BaseModel.php on line 762.
 1 SYSTEMPATH\BaseModel.php(762): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([], true)
 3 APPPATH\Controllers\DevisController.php(54): CodeIgniter\Model->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DevisController->insert_devis()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
ERROR - 2023-04-30 23:03:33 --> mysqli_sql_exception: Unknown column 'devis.deleted_at' in 'where clause' in C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\ci4\project-root\app\Controllers\DevisController.php(22): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\DevisController->index()
#8 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
#9 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4\project-root\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2023-04-30 23:03:33 --> Unknown column 'devis.deleted_at' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `devis`
WHERE `devis`.`deleted_at` IS NULL', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `devis`
WHERE `devis`.`deleted_at` IS NULL')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `devis`
WHERE `devis`.`deleted_at` IS NULL')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `devis`
WHERE `devis`.`deleted_at` IS NULL', [], false)
 5 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\DevisController.php(22): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DevisController->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
ERROR - 2023-04-30 23:05:03 --> mysqli_sql_exception: Unknown column 'devis.deleted_at' in 'where clause' in C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\ci4\project-root\app\Controllers\DevisController.php(22): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\DevisController->index()
#8 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
#9 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4\project-root\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2023-04-30 23:05:03 --> Unknown column 'devis.deleted_at' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `devis`
WHERE `devis`.`deleted_at` IS NULL', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `devis`
WHERE `devis`.`deleted_at` IS NULL')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `devis`
WHERE `devis`.`deleted_at` IS NULL')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `devis`
WHERE `devis`.`deleted_at` IS NULL', [], false)
 5 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\DevisController.php(22): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DevisController->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
ERROR - 2023-04-30 23:05:10 --> mysqli_sql_exception: Unknown column 'devis.deleted_at' in 'where clause' in C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\ci4\project-root\app\Controllers\DevisController.php(22): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\DevisController->index()
#8 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
#9 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4\project-root\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4\project-root\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2023-04-30 23:05:10 --> Unknown column 'devis.deleted_at' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `devis`
WHERE `devis`.`deleted_at` IS NULL', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `devis`
WHERE `devis`.`deleted_at` IS NULL')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `devis`
WHERE `devis`.`deleted_at` IS NULL')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `devis`
WHERE `devis`.`deleted_at` IS NULL', [], false)
 5 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\DevisController.php(22): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DevisController->index()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DevisController))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4\\project-root\\public\\index.php')
